portal-wire-protocol.md

```diff
@@ -126,24 +121,22 @@ Response message to FindNodes(0x02).
 
 ```
 selector     = 0x03
-nodes        = Container(total: uint8, enrs: List[ByteList, max_length=32])
+nodes        = Container(total: uint8, enrs: List[ByteList[2048], max_length=32])
 ```
 
-* `total`: The total number of `Nodes` response messages being sent.
+* `total`: The total number of `Nodes` response messages being sent. Currently fixed to only 1 response message.
```

Comment:
I think this is an important call out that we haven't discussed in quite a while.  Thanks for highlighting.